panel/config/addplugindialog.cpp

```diff
@@ -34,10 +34,13 @@
 #include <XdgIcon>
 #include <XdgDirs>
 
-#include <QString>
+#include <QIcon>
 #include <QLineEdit>
 #include <QListWidgetItem>
-#include <QIcon>
+#include <QString>
+#include <chrono>
+
+using namespace std::chrono_literals;
```

Comment:
Any reason for using `chrono`? `QTimer::setInterval(int msec)` exists and `QTimer::setInterval(std::chrono::milliseconds value)` is an overloaded function.

Apart from thai, IMHO, it's highly preferable not to change a code while porting, unless really necessary.

Reply:
Hi, you don't need to search all references. It's all in commit b5474baff73f6d88bb63b4d45f6975efcbb5731a

It was automatically done by Clazy and I think this way it's far more readable, but I can move the commit is a separate PR after Qt6 port.

Reply:
> but I can move the commit in a separate PR

Let me explain it pragmatically, from a reviewer's perspective; it might convince you if you put yourself in my shoes.

Porting to a new version of Qt may seem straightforward in most places, but (1) the amount of changes is heavy, and (2) there's always the danger of introducing regressions, especially where the new version of Qt shows an undocumented different behavior (which has happened  in my port of libfm-qt). To avoid such regressions as much as possible, we need to stick with the current code during porting and change them only if *really* needed, such that **the changed codes reflect what is really needed to be done** (there are several examples of that in https://github.com/lxqt/libfm-qt/pull/778/files).

As for a component like libfm-qt or pcmanfm-qt, I've used, tested and fixed its Qt6 port continuously for more than a year. Sadly, we won't have that time for the remaining components. Hence, the less unnecessary changes, the better.